dictionary_client/utils.py

```diff
@@ -0,0 +1,25 @@
+import re
```

Comment:
Could you add a module level docstring pointing to the grammar ( https://datatracker.ietf.org/doc/html/rfc2229#section-2.2 )? I think that will help with grokking the regular expressions.

Reply:
Added!